nucliadb_ingest/nucliadb_ingest/orm/resource.py

```diff
@@ -166,9 +166,28 @@ async def set_basic(self, payload: PBBasic, slug: Optional[str] = None):
         await self.get_basic()
         if self.basic is not None and self.basic != payload:
             self.basic.MergeFrom(payload)
+
             # We force the usermetadata classification to be the one defined
             if payload.HasField("usermetadata"):
                 self.basic.usermetadata.CopyFrom(payload.usermetadata)
+
+            if len(payload.fieldmetadata):
+                # keep only the last fieldmetadata item for each field. API
+                # users are responsible to manage fieldmetadata properly
```

Comment:
WDYT about adding a test with this case (modify the same field twice on the same payload)? Just to be even more explicit.